src/Routing/Duplex.purs

```diff
@@ -64,9 +71,12 @@ instance applicativeRouteDuplex :: Applicative (RouteDuplex i) where
 instance profunctorRouteDuplex :: Profunctor RouteDuplex where
   dimap f g (RouteDuplex enc dec) = RouteDuplex (f >>> enc) (g <$> dec)
 
+-- | Use given codec to parse a value of type `o` out of String
+-- | (typically representing part of URL) or produce a `RouteError` if the parsing fails.
 parse :: forall i o. RouteDuplex i o -> String -> Either Parser.RouteError o
 parse (RouteDuplex _ dec) = Parser.run dec
 
+-- | Use given codec to transform a value of type `i` to a String.
```

Comment:
Maybe instead of "transform to a String", we can say that it prints the `i` type to a URL path, query parameters, and hash. Kind of like with `parse`, since it's just the inverse.